src/screens/Inventory/components/PurchaseRequestCard.tsx

```diff
@@ -11,56 +15,92 @@ import {
   selectProductById
 } from '../../../lib/redux/inventoryDataSlice';
 import { RootState } from '../../../lib/redux/store';
+import { selectCurrentUserId, selectCurrentUserIsAdmin } from '../../../lib/redux/userData';
+import { reviewPurchaseRequest } from '../../../lib/utils/inventoryUtils';
+import { getPurchaseRequestReviewButtons } from '../PurchaseRequest';
 
 const styles = (theme: Theme) =>
   createStyles({
-    arrow: {
-      color: theme.palette.text.primary,
-    },
     cardContent: {
       flex: 1,
-      padding: 16,
+      alignItems: 'center',
+      display: 'flex',
+      paddingRight: theme.spacing(1),
+      '&:last-child': {
+        paddingBottom: theme.spacing(2),
+      },
+    },
+    leftContent: {
+      flex: 1,
+      marginRight: theme.spacing(1),
     },
     cardContainer: {
-      border: `1px solid ${theme.palette.divider}`,
-      boxShadow: 'none',
-      borderRadius: 10,
+      borderRadius: 6,
       display: 'flex',
-      marginBottom: 10,
+      marginBottom: theme.spacing(1),
     },
-    arrowSpacing: {
-      padding: 0,
+    cardActions: {
+      width: 70,
```

Comment:
More of a question for me so I understand styling better: I realize in this case the pixel width is to align the icons and the text consistently (when there's 1 icon vs 2 icon). When is it generally appropriate to set a pixel width/height?

Reply:
Good question. In most cases you shouldn't need to use a fixed px width/height but I had to here so that the right-most column would line up no matter how many icons there are. HONESTLY it was very painful for me to have to resort to this 😢  but I couldn't think of a better way (besides adding a transparent button of the same width ??)
![image](https://user-images.githubusercontent.com/21160510/114810204-6a410780-9d60-11eb-850b-2ff849d68e23.png)